mutator/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/mutator/TypeReceiver.scala

```diff
@@ -102,9 +102,9 @@ object TypeReceiver {
   def apply(queue: Queue[IO, List[ShreddedType]], verbose: Boolean): TypeReceiver =
     new TypeReceiver(queue, verbose)
 
-  def startSubscription(config: Config, listener: TypeReceiver): IO[Unit] =
+  def startSubscription(env: MutatorEnvironment, listener: TypeReceiver): IO[Unit] =
     IO {
-      val subscription = ProjectSubscriptionName.of(config.projectId, config.typesSubscription)
+      val subscription = ProjectSubscriptionName.of(env.projectId, env.config.input.subscription)
       val subscriber   = Subscriber.newBuilder(subscription, listener).setHeaderProvider(UserAgent).build()
       subscriber.startAsync().awaitRunning(10L, TimeUnit.SECONDS)
```

Comment:
I know nothing about this subscriber or what it does 😉 but if it is awaiting something something for 10 seconds, then I would expect to see a cats effect `Blocker` being used.  I know you didn't write this line of code so it's fine if you want to leave it.

Reply:
I think it uses own threadpool underneath, so we probably had to use `IO.async`